changelog.config.js

```diff
@@ -0,0 +1,79 @@
+const COMPONENT_SPECS = require('genesys-spark-components/build/component-specs/component-specs.json');
+const CHART_COMPONENT_SPECS = require('genesys-spark-chart-components/build/component-specs/component-specs.json');
+
+const componentSpecs = Object.assign(COMPONENT_SPECS, CHART_COMPONENT_SPECS);
+
+const getComponentSpec = function () {
+  let scopeList = ['other'];
```

Comment:
not worth a change but technically this can be a `const` as you never assign a new value to the variable it is always the same array and you are pushing values into the array